src/pages/workspace/accounting/xero/XeroMapTrackingCategoryConfigurationPage.tsx

```diff
@@ -1,61 +1,67 @@
+import {useRoute} from '@react-navigation/native';
 import React, {useCallback, useMemo} from 'react';
 import ConnectionLayout from '@components/ConnectionLayout';
 import SelectionList from '@components/SelectionList';
 import RadioListItem from '@components/SelectionList/RadioListItem';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as Connections from '@libs/actions/connections';
-import {getTrackingCategory} from '@libs/actions/connections/ConnectToXero';
 import Navigation from '@libs/Navigation/Navigation';
 import type {WithPolicyProps} from '@pages/workspace/withPolicy';
 import withPolicyConnections from '@pages/workspace/withPolicyConnections';
 import CONST from '@src/CONST';
 import type {TranslationPaths} from '@src/languages/types';
 import ROUTES from '@src/ROUTES';
 
-function XeroMapCostCentersToConfigurationPage({policy}: WithPolicyProps) {
+type RouteParams = {
+    categoryId?: string;
+    categoryName?: string;
+};
+
+function XeroMapTrackingCategoryConfigurationPage({policy}: WithPolicyProps) {
     const {translate} = useLocalize();
+    const route = useRoute();
+    const params = route.params as RouteParams;
     const styles = useThemeStyles();
-
+    const categoryId = params?.categoryId ?? '';
+    const categoryName = params?.categoryName ?? '';
```

Comment:
Since we encode the `categoryName` in our route, we should decode it here to make sure we get the correct value for it.
```suggestion
    const categoryName = decodeURIComponent(params?.categoryName ?? '');
```

Reply:
I do agree, it's cleaner that way.